src/main/java/com/google/api/generator/engine/ast/TypeNode.java

```diff
@@ -125,6 +135,14 @@ public static boolean isReferenceType(TypeNode type) {
         && !type.equals(TypeNode.NULL);
   }
 
+  public static boolean isNumericType(TypeNode type) {
+    return NUMERIC_TYPE_SET.contains(type);
+  }
+
+  public static boolean isBoxedType(TypeNode type) {
```

Comment:
Could you please confirm if this is still needed? `isNumericType` will implicitly call equals.

Reply:
I see your point. I removed the set to look up and leveraged equals in `IsNumericType`, removed `isBoxedType`